src/main/java/use_cases/DataAccess.java

```diff
@@ -0,0 +1,24 @@
+package use_cases;
+
+import entities.HumanPlayer;
+
+import java.io.IOException;
+
+public interface DataAccess {
+    /**
+     * Loads a HumanPlayer's wins and losses from "src/main/java/use_cases/players.csv" given their name.
+     * @param name The name of the HumanPlayer.
+     * @return A string array [name, wins, losses] of the HumanPlayer, or null if the name is not found.
+     * @throws IOException If an input/output error occurs when reading from the file.
+     */
+    String[] loadPlayer(String name) throws IOException;
```

Comment:
i think a data class instead of a string array is better for this return value

Reply:
Got it, created PlayerInformation class.